source/jsonnet/northern-ireland/individual/blocks/identity-and-health/national_identity.jsonnet

```diff
@@ -49,12 +46,7 @@ local question(title, detailAnswerLabel) = {
         {
           label: 'Other',
           value: 'Other',
-          detail_answer: {
-            id: 'national-identity-answer-other',
-            type: 'TextField',
-            mandatory: false,
-            label: detailAnswerLabel,
-          },
+          description: otherDescription,
```

Comment:
In Overflow `Other` option doesn't have a description anymore. Is this part of your card?

Reply:
Yeah, Overflow is out of date, the spreadsheet is the source of truth and does have it. The  'of' is a mistake and does need changing